tools/tenscan/frontend/src/components/modules/dashboard/recent-transactions.tsx

```diff
@@ -17,6 +17,11 @@ export function RecentTransactions({ transactions }: { transactions: any }) {
                 #{transaction?.BatchHeight}
               </p>
             </div>
+            <div className="ml-4 space-y-1">
+              <p className="text-sm font-medium leading-none">
+                {transaction?.BatchTimestamp}
```

Comment:
Would it work to use `{formatTimeAgo(batch?.timestamp)}` here, like @Jennievon did in `recent-batches.tsx`? To get the 'x mins ago' formatting.